openapi.yml

```diff
@@ -86,6 +86,20 @@ paths:
         204:
           description: No content
   /organizations:
+    get:
+      description: |-
+        Returns all the organizations related to the user
+      operationId: getOrganizations
+      security:
+        - bearerAuth: []
+      responses:
+        <<: *common-responses
+        200:
+          description: The organizations related to the user
+          content:
+            application/json:
+              schema:
+                $ref: "#/components/schemas/GetOrganizationResults"
```

Comment:
I won't repeat the http method in the schema name, eg `GetOrganization` -> `Organization`